### PR TITLE
ROSTESTS-316. Improve WndProc ApiTest

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

This PR is only improving current code.
It does not target the main issue.

JIRA issue: [ROSTESTS-316](https://jira.reactos.org/browse/ROSTESTS-316)
WTB result: [Job 42474](https://testbot.winehq.org/JobDetails.pl?Key=42474)

### CODE REVIEW

File: modules/rostests/apitests/user32/WndProc.c [hunk 1]
```diff
@@ -9,96 +9,143 @@
...
             EndPaint(hwnd, &ps);
             return 1;
         }
-   /*
-       This will force one stack corruption "ret" fault with normal window
-       procedure callback.
-    */
+
+        /*
+         * This will force one stack corruption "ret" fault with normal window
```
Comment (learn-more) [inline modules/rostests/apitests/user32/WndProc.c]: Is this a new comment style?

Comment (SergeGautherie) [reply modules/rostests/apitests/user32/WndProc.c]: Hum. You wanted `//`s, right?

Comment (learn-more) [reply modules/rostests/apitests/user32/WndProc.c]: No, but adding '*' in front of all lines is not needed at all.

File: modules/rostests/apitests/user32/WndProc.c [hunk 2]
```diff
@@ -9,96 +9,140 @@
...
+                 "ret");
 #elif defined(_M_IX86)
 //#ifdef _MSC_VER
-        __asm
-          {
-             mov eax, 0
-             leave
-             ret
-          }
+            trace("Executing _M_IX86 stack corruption code\n");
```
Comment (tkreuzer) [inline modules/rostests/apitests/user32/WndProc.c]: Better "Executing MSVC x86 stack corruption code\n"

Comment (SergeGautherie) [reply modules/rostests/apitests/user32/WndProc.c]: Done.
Should I s/__MINGW32__/GCC x86/ too?

**Pull request merged**